SpringContainerMS/chart/springcontainerms/templates/deployment.yaml

```
@@ -69,6 +69,12 @@ spec:
                 name: "{{ .Values.bpm.anomalyConfigMap }}"
                 key: login
                 optional: true
+          - name: BPM_EXPIRATION
+            valueFrom:
+              secretKeyRef:
```

Need to change this to `configMapKeyRef`- 

```suggestion
              configMapKeyRef:
```